<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateConfiguration</class>
 <widget class="QMainWindow" name="GenerateConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="labelRelay">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>Przekaźniki</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="tableWidget">
      <property name="minimumSize">
       <size>
        <width>734</width>
        <height>398</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>734</width>
        <height>398</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>4</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>5</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>6</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>7</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>8</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>9</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>10</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>11</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>12</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelSpace">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonGenerate">
        <property name="text">
         <string>Generuj</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBack">
        <property name="text">
         <string>Powrót</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="line1">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line2">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line3">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line4">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line5">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line6">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line7">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line8">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line9">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line10">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line11">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line12">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="line13">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>33</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line14">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>33</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line15">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>33</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line16">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>33</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line17">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>33</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line18">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>33</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line19">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>33</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line20">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>33</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line21">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>33</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line22">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>33</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line23">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>33</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line24">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>33</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
